src/EventInterface.php

```diff
@@ -1,14 +1,54 @@
 <?php
 
-namespace Dhii\Event;
-
-use Psr\EventManager\EventInterface as BaseEventInterface;
+namespace Dhii\Events\Event;
 
 /**
  * Represents an application event.
  *
  * @since [*next-version*]
  */
-interface EventInterface extends BaseEventInterface
+interface EventInterface
 {
+    /**
+     * Get event name.
+     *
+     * @since [*next-version*]
+     *
+     * @return string The name of the event.
+     *                The event name contains mixed-case alphanumeric characters, as well as '_' (underscore) and '.' (period).
```

Comment:
What about '/'?

Reply:
This spec is taken directly from the previous draft PSR-14 version that we had mirrored and you had implemented. Do you think we should include the `'/'` there? I'd say, let's leave it for now as it was, and add if needed. I would like to think that FIG excluded this character for a reason.

Reply:
Agreed. I only asked in case this _wasn't_ from PSR-14. Let's leave it for now.